addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -88,11 +89,40 @@ class MessageList extends Component {
                 threadViewer: threadView && threadView.threadViewer,
             };
         });
+        /**
+         * This observer will emit an hit when the composer is resized by any
+         * mean.
+         */
+        this.resizeObserver = new ResizeObserver(elements => {
+            // In this case, this loop seems useless since we are really working
+            // on a single element, but it's required by the ResizeObserver to
+            // avoid error: ResizeObserver loop completed with undelivered
+            // notifications.
+            for (const element of elements) {
+                if (this.threadView &&
+                    this.threadView.thread &&
+                    this.threadView.thread.chatWindow &&
+                    this.threadView.thread.chatWindow.isBottom
+                   ) {
+                    this._adjustScrollForExtraMessagesAtTheEnd();
+                }
+            }
+        });
         // useUpdate must be defined after useRenderedValues to guarantee proper
         // call order
         useUpdate({ func: () => this._update() });
     }
 
+    mounted() {
+        if (this.props.order === 'asc') {
+            this.resizeObserver.observe(this._messageList.el);
+        }
+    }
```

Comment:
resizing after OWL re-render happens in ASC configuration right now, but I think it's safe to apply whatever the configuration. If we add some features that mess up message list sizing also in DESC, would be nice that it simply works than having to adapt this LOC... Especially when it will not be obvious at all in 2 of 3 months from now.

Reply:
Alright !
I wasn't confident enough to not add this guard. But since I couldn't find any side effect, I guess it's ok to remove it.